test/shell/prometheus-2.sh

```diff
@@ -2,17 +2,39 @@
 
 . "$(dirname "$0")/../util.sh"
 
+TEMPFILE=`mktemp mgmt-tests-XXXXXXXX`
+
+cleanup()
```

Comment:
I'm fine with having these here, but would putting them in the above listed util.sh help, and/or is there something there that's already useful?